### DISCUSSION TITLE
Add ID to Row Deleted Trigger Output

### DISCUSSION THREAD

**Discussion by bakert-96** [Budibase/budibase#4335] [Ideas]
Hello!

I would like to propose the idea of having the ID of the row deleted be available in the Row Deleted trigger output. AFAIK it's not available in the Row Deleted trigger, but is in the Row Updated and Row Created triggers. I made this change in a local dev environment to prototype something where I import a CSV into Budibase table, then use automation triggers to insert/update/delete that data in Elasticsearch and seems to all be working good. I did have to change one other thing in the delete function in elasticsearch.ts as it was giving me a 400 invalid argument exception -- seems to possibly be due to the customData key in the query object but haven't fully looked through that yet and would probably be a separate discussion.

**Comment by mjashanks** [comment]
Sounds good! I agree having the ID would be very useful - surprised we don't have it. If you have a working PR - please submit :)
